jbpm-benchmarks/jbpm-performance-tests-jmh/src/main/java/org/jbpm/test/performance/scenario/load/LHumanTaskProcess.java

```diff
@@ -35,13 +37,34 @@
     @Param("")
     public String runtimeManagerStrategy;
     private JBPMController jc;
+    RuntimeManager manager;
 
-    @Setup
+    @State(Scope.Thread)
+    public static class ThreadScope {
+
+        public RuntimeEngine runtimeEngine;
+        public KieSession ksession;
+        JBPMController jc;
+        RuntimeManager manager;
```

Comment:
I think we need only `RuntimeEngine` here in the thread scope. As in the teardown we need to call only `manager.disposeRuntimeEngine(runtimeEngine)`. Everything else is redundant.